scripts/ec2cleaning.py

```diff
@@ -0,0 +1,40 @@
+from datetime import datetime
+from utils.providers import list_providers, get_mgmt
+from time import sleep
+date = datetime.now()
+deletetime = 60 * 60 * 24 * 7  # 7 Days
+
+
+def delete_old_instances(ec2provider, date, deletetime):
+    for instance in ec2provider.list_vm(include_terminated=True):
```

Comment:
@mmojzis It would be nice to handle the exceptions and log erros in all these methods, it makes it easy to track the script errors.